ably/realtime_conn.go

```diff
@@ -413,6 +414,9 @@ func (c *Connection) connectWith(arg connArgs) (result, error) {
 func (c *Connection) connectionStateTTL() time.Duration {
```

Comment:
@tcard this function returns `connectionStateTTL` based on default value passed as 1. arg 2. options or 3. from connected message